src/scss/customOverrides.scss

```
@@ -10,3 +11,20 @@
     display: none;
   }
 }
+
+// hide glow from Auditorium v1 iframe
+.audience-container {
+  .audience-overlay {
+    .video-container {
+      .reaction-container {
+        box-shadow: none;
+      }
+      .video {
```

Spacing between sibling blocks for readability:
```suggestion
      }

      .video {
```